src/components/Calculator.js

```diff
@@ -0,0 +1,35 @@
+import './Calculator.css';
+
+function Wrapper() {
```

Comment:
- [ ] It would be better if your calculator is class-based as one of the requirements for this milestone. Kindly fix this minor issue. 

😄 💪 

```suggestion
```javascript
class Calculator extends Component {
  render() {
```
```

Reply:
thanks a lot for your review @BrianSammit. Making the component as a react component in the App.js file does not work ?

Reply:
The component works without problem. In future milestones, you will change the class-bases component but this for milestone must be class_based.  👍 😄